### ISSUE TITLE
[CoE Starter Kit - BUG] Unmanage layers kept re-appearing and unable to remove the unmanaged layers

### ISSUE THREAD

**Issue by faszul** [microsoft/coe-starter-kit#7598]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

I'm currently performing CoE upgrade prep and one of the crucial steps is to remove the unmanaged layers and this is advised in case #6470.

After the removal and waiting for the flows to run again then unmanaged flows re-appear and when I tried again it shows an error attached inside this case.

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

1.4.6070-2307.1

### What app or flow are you having the issue with?

Admin Compliance Detail Request

### What method are you using to get inventory and telemetry?

None

**Comment by RahulBojji** [comment]
@faszul 

Can you please share the error you are facing, looks like you have missed it? If you are trying to turn it on, please remove the unmanaged layers and try to turn it using setup wizard. It might not create unmanaged layers.
Go to COE Setup wizard -> More Features -> Compliance Process

<img width="803" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/149372606/728dac8c-aaba-4d28-8c0b-17c7b38c91a2">

**Comment by faszul** [comment]
Hi, this is the error when I tried to remove the layer.
![Image 06-02-2024 at 4 36 PM](https://github.com/microsoft/coe-starter-kit/assets/142005629/8a281233-c95d-4007-8381-2cc0bc9f2780)

**Comment by Jenefer-Monroe** [comment]
Unfortunately you are hitting product issues, not issues with the kit. But we can try a little more to get you unblocked.
Does this user id correctly correspond to the id of the user that is logged in trying to remove the layer?
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/f91c67d8-b835-44e9-bcad-94bd817e5233)

**Comment by faszul** [comment]
Hi Jennifer,

I’m logging in as my Office365 admin ID but the ID in the screenshot is not the same. I tried to logged out and re-signed in and got the same error.

Regards,
Fahdzul
[Image]


From: Jenefer Monroe ***@***.***>
Date: Saturday, 10 February 2024 at 12:51 AM
To: microsoft/coe-starter-kit ***@***.***>
Cc: Aziz, Fahdzul ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [microsoft/coe-starter-kit] [CoE Starter Kit - BUG] Unmanage layers kept re-appearing and unable to remove the unmanaged layers (Issue #7598)
Unfortunately you are hitting product issues, not issues with the kit. But we can try a little more to get you unblocked. Does this user id correctly correspond to the id of the user that is logged in trying to remove the layer? image. png (view


Unfortunately you are hitting product issues, not issues with the kit. But we can try a little more to get you unblocked.
Does this user id correctly correspond to the id of the user that is logged in trying to remove the layer?
image.png (view on web)<https://urldefense.com/v3/__https:/github.com/microsoft/coe-starter-kit/assets/89584709/f91c67d8-b835-44e9-bcad-94bd817e5233__;!!MfzFaTml5A!mrGXdNKv7W1ambfukSgVyDTrxBAyuvyC3gZiIv7m3gCy9QLBQNppAt0j-hR4AZ29K3Evwth9bGW8pJIxF_xma4PU2ICeTUMcNNEMrMif$>

**Comment by Jenefer-Monroe** [comment]
Yes a few people have been hitting this lately where the wrong ID is getting cached. You'll need to clear your browser cache and then hopefully it will log you in as the correct person.

**Comment by faszul** [comment]
Hi Jenefer,

Tried different browser and cleared the cache but I'm hitting the same error. Is there any other way to purge the unmanage layers? Do take note that I still can perform any other CoE functions.

**Comment by Jenefer-Monroe** [comment]
You can manually remove unmanaged layers as shown here: [Removing other customizations](https://learn.microsoft.com/power-platform/guidance/coe/after-setup#removing-other-customizations)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
